src/devices/AD5328/samples/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using System.Drawing;
+using System.Threading;
+using Iot.Device.AD5328;
+
+var spisettings = new SpiConnectionSettings(0, 1)
+{
+    Mode = SpiMode.Mode2
+};
+
+var spidev = SpiDevice.Create(spisettings)
```

Comment:
you are missing a `;` at the end. Please correct the documentation as well.
Tip: at the root of the project you cloned, run build.cmd or build.sh depending of your OS, this will catch those type of errors.

Reply:
Thank you for the tips, sorry about that. Should be fixed now.